CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project. You can start setting up with these next steps.
+
+1. Anyone who would like to contribute to **[Code.org](https://github.com/code-dot-org/)** projects **must read and sign the [Contributor License Agreement](https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=8eb90665-c9f7-4b06-81a5-11d12020f251)**. We can't accept pull requests from contributors who haven't yet signed the CLA.
+
+2. Get your local development build working by following our [README](README.md). Fork our repo and make sure to merge our staging branch into yours **WEEKLY** as we do update frequently.
+
+3. [Join our community development HipChat room](http://www.hipchat.com/gBebkHP6g) for help getting set up. We're happy to have you! If you want to make sure you get our attention, include an **@all** (everyone) or **@here** (everyone currently in the room) in your message.
+
+## Code Style
+
+Please make sure your code conforms to our [style guide](STYLEGUIDE.md).  We have implemented linting to help you with this:
+
+* To install our precommit hook that will lint the files you've changed, run `rake install:hooks`. You should only have to do this once.  We _highly_ recommend it!
```

Comment:
Can we make this part of regular rake:install?

Also, don't _highly_recommend something, just say it's required or their build will fail whatever hound checks.